tests/listsAndDropDowns.spec.ts

```diff
@@ -9,76 +9,76 @@ test.beforeEach(async ({ page }) => {
 
 test.describe("Lists and Drop Downs", async () => {
   test("Validate selected pet types from the list", async ({ page }) => {
+    const type1Input = page.locator("#type1");
+    const petTypeSelect = page.locator("#type");
     await page.getByText("George Franklin").click();
 
     await expect(page.locator(".ownerFullName")).toHaveText("George Franklin");
 
-    await page.getByRole("button", { name: "Edit Pet" }).nth(0).click();
+    const petLeoSection = page.locator("app-pet-list", { name: "Leo" });
+    await petLeoSection
+      .getByRole("button", { name: "Edit Pet" })
+      .first()
```

Comment:
Index `first()` is not needed here, because Leo has only one `Edit Pet` button

Reply:
done